packages/orbit-components/src/Tabs/components/Tab/types.d.ts

```diff
@@ -0,0 +1,11 @@
+import type { Globals } from "../../../common/types";
+import type { TYPE_OPTIONS } from "./consts";
+
+export type Type = Lowercase<keyof typeof TYPE_OPTIONS>;
```

Comment:
I believe this can be simplified into
```suggestion
export type Type = `${TYPE_OPTIONS}`;
```

Reply:
yeah, forgot about that, thanks ;)